README.md

```diff
@@ -156,6 +156,13 @@ cargo test $TESTNAME
 
 Our CI systems automatically test all the pull requests, so making sure the full suite passes the test before creating your PR is not strictly required. **All merged PRs must have passed CI test.**
 
+Note that, to reduce compilation time, TiKV builds do not include debugging information by default. The easiest way to enable debuginfo is to procede build commands with `RUSTFLAGS=-Cdebuginfo=true`.
+
+```bash
+RUSTFLAGS=-Cdebuginfo=true make
```

Comment:
```suggestion
RUSTFLAGS=-Cdebuginfo=1 make
```

Reply:
Can you say more about why you suggest using "1" here?

Reply:
Sorry I didn't make it clear, rustc reports errors:

```
sh-4.2$ rustc --version
rustc 1.36.0-nightly (e305df184 2019-04-24)
sh-4.2$ RUSTFLAGS=-Cdebuginfo=true cargo build
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `/home/root/.cargo/bin/sccache rustc - --crate-name ___ --print=file-names -Cdebuginfo=true --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro` (exit code: 1)
--- stderr
error: incorrect value `true` for codegen option `debuginfo` - a number was expected


sh-4.2$ RUSTFLAGS=-Cdebuginfo=1 cargo build
   Compiling cfg-if v0.1.6
```

Reply:
lol that's a good reason. I'm not sure what I was thinking. Maybe cargo supports true/false and translates them to numbers.